src/main/java/io/github/bedwarsrel/BedwarsRel/Shop/Specials/ProtectionWallListener.java

```diff
@@ -79,6 +79,7 @@ public void onPlace(BlockPlaceEvent place) {
 
     place.setBuild(false);
     place.setCancelled(true);
+    return;

```

Comment:
Naja... Es sollte verhindern, dass es zu einem blockplace kommt. In meinen Tests hat geholfen. Aber auch vorher blieb der Block nur selten.